Move prover returns: exiting with boogie verification errors
error: abort not covered by any of the `aborts_if` clauses

    ┌── tests/sources/functional/loops.move:86:5 ───
    │
 86 │ ╭     spec iter10_abort_incorrect { // Disproved. Abort always happens.
 87 │ │         pragma verify=true;
 88 │ │         aborts_if false;
 89 │ │     }
    │ ╰─────^
    ·
 82 │             if (i == 7) abort 7;
    │                         ------- abort happened here with code 0x7
    │
    =     at tests/sources/functional/loops.move:77: iter10_abort_incorrect
    =         i = <redacted>
    =     at tests/sources/functional/loops.move:79: iter10_abort_incorrect
    =     at tests/sources/functional/loops.move:80: iter10_abort_incorrect
    =     at tests/sources/functional/loops.move:78: iter10_abort_incorrect
    =     at tests/sources/functional/loops.move:82: iter10_abort_incorrect
    =         ABORTED

error: function does not abort under this condition

    ┌── tests/sources/functional/loops.move:58:9 ───
    │
 58 │         aborts_if true;
    │         ^^^^^^^^^^^^^^^
    │
    =     at tests/sources/functional/loops.move:48: iter10_no_abort_incorrect
    =         i = <redacted>
    =     at tests/sources/functional/loops.move:50: iter10_no_abort_incorrect
    =     at tests/sources/functional/loops.move:51: iter10_no_abort_incorrect
    =     at tests/sources/functional/loops.move:49: iter10_no_abort_incorrect
    =     at tests/sources/functional/loops.move:58

error: base case of the loop invariant does not hold

     ┌── tests/sources/functional/loops.move:210:17 ───
     │
 210 │                 assert x != 0;
     │                 ^^^^^^^^^^^^^^
     │
     =     at tests/sources/functional/loops.move:206: loop_invariant_base_invalid
     =         n = <redacted>
     =     at tests/sources/functional/loops.move:207: loop_invariant_base_invalid
     =         x = <redacted>
     =     at tests/sources/functional/loops.move:209: loop_invariant_base_invalid
     =     at tests/sources/functional/loops.move:210: loop_invariant_base_invalid

error: induction case of the loop invariant does not hold

     ┌── tests/sources/functional/loops.move:223:17 ───
     │
 223 │                 assert x == 0;
     │                 ^^^^^^^^^^^^^^
     │
     =     at tests/sources/functional/loops.move:219: loop_invariant_induction_invalid
     =         n = <redacted>
     =     at tests/sources/functional/loops.move:220: loop_invariant_induction_invalid
     =         x = <redacted>
     =     at tests/sources/functional/loops.move:222: loop_invariant_induction_invalid
     =     at tests/sources/functional/loops.move:223: loop_invariant_induction_invalid
     =     at tests/sources/functional/loops.move:225: loop_invariant_induction_invalid
     =     at tests/sources/functional/loops.move:221: loop_invariant_induction_invalid
     =         x = <redacted>
     =     at tests/sources/functional/loops.move:223: loop_invariant_induction_invalid

error: induction case of the loop invariant does not hold

     ┌── tests/sources/functional/loops.move:189:17 ───
     │
 189 │                 assert x != y;
     │                 ^^^^^^^^^^^^^^
     │
     =     at tests/sources/functional/loops.move:183: loop_with_two_back_edges_incorrect
     =         x = <redacted>
     =         y = <redacted>
     =     at tests/sources/functional/loops.move:185: loop_with_two_back_edges_incorrect
     =     at tests/sources/functional/loops.move:188: loop_with_two_back_edges_incorrect
     =     at tests/sources/functional/loops.move:189: loop_with_two_back_edges_incorrect
     =     at tests/sources/functional/loops.move:191: loop_with_two_back_edges_incorrect
     =         x = <redacted>
     =     at tests/sources/functional/loops.move:197: loop_with_two_back_edges_incorrect
     =     at tests/sources/functional/loops.move:189: loop_with_two_back_edges_incorrect

error: base case of the loop invariant does not hold

     ┌── tests/sources/functional/loops.move:145:21 ───
     │
 145 │                     assert x != y;
     │                     ^^^^^^^^^^^^^^
     │
     =     at tests/sources/functional/loops.move:138: nested_loop_inner_invariant_incorrect
     =         x = <redacted>
     =         y = <redacted>
     =     at tests/sources/functional/loops.move:140: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:143: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:144: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:145: nested_loop_inner_invariant_incorrect

error: induction case of the loop invariant does not hold

     ┌── tests/sources/functional/loops.move:145:21 ───
     │
 145 │                     assert x != y;
     │                     ^^^^^^^^^^^^^^
     │
     =     at tests/sources/functional/loops.move:138: nested_loop_inner_invariant_incorrect
     =         x = <redacted>
     =         y = <redacted>
     =     at tests/sources/functional/loops.move:140: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:143: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:144: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:145: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:147: nested_loop_inner_invariant_incorrect
     =     at tests/sources/functional/loops.move:150: nested_loop_inner_invariant_incorrect
     =         y = <redacted>
     =     at tests/sources/functional/loops.move:145: nested_loop_inner_invariant_incorrect

error: induction case of the loop invariant does not hold

     ┌── tests/sources/functional/loops.move:119:17 ───
     │
 119 │                 assert x != y;
     │                 ^^^^^^^^^^^^^^
     │
     =     at tests/sources/functional/loops.move:113: nested_loop_outer_invariant_incorrect
     =         x = <redacted>
     =         y = <redacted>
     =     at tests/sources/functional/loops.move:115: nested_loop_outer_invariant_incorrect
     =     at tests/sources/functional/loops.move:118: nested_loop_outer_invariant_incorrect
     =     at tests/sources/functional/loops.move:119: nested_loop_outer_invariant_incorrect
     =     at tests/sources/functional/loops.move:122: nested_loop_outer_invariant_incorrect
     =         x = <redacted>
     =     at tests/sources/functional/loops.move:119: nested_loop_outer_invariant_incorrect
